### ISSUE TITLE
Bad `beta` versioning

### ISSUE THREAD

**Issue by Vovan-VE** [jpuri/html-to-draftjs#32]
Version must be `...-beta.N` but not `...-betaN`.
Currently `0.1.0-beta15` < `0.1.0-beta9`.

```js
> semver.lt('0.1.0-beta15', '0.1.0-beta9')
true
> semver.gt('0.1.0-beta15', '0.1.0-beta9')
false
> semver.gt('0.1.0-beta.15', '0.1.0-beta.9')
true
```

```
"html-to-draftjs": "^0.1.0-beta15"
```

```sh
$ npm outdated
Package                   Current         Wanted        Latest  Location
html-to-draftjs      0.1.0-beta15    0.1.0-beta9  0.1.0-beta15 ...
```

So, `npm update` actually does downgrade to `beta9`.

**Comment by jpuri** [comment]
Thanks a lot for bringing this up @Vovan-VE , I just published `html-to-draftjs@0.1.0-beta.16` today :)

**Comment by Vovan-VE** [comment]
Good step. But it's not so simple. Name `beta` < `betaWHATEVER`, so `-beta.*` < `-beta15` < `-beta9`.
To solve the problem you can use anything alphabetically greater then `beta`. For example, `rc`, `pre`, or even `betb`. Also `beta90` > `beta9` > `beta15`. In this case version will be like `-beta90.16`

**Comment by jpuri** [comment]
Just released `1.0.0` today. The lib is pretty stable now.
